driver-core/pom.xml

```diff
@@ -334,6 +334,13 @@
               <classpathDependencyExcludes>
                 <classpathDependencyExcludes>io.netty:netty-transport-native-epoll</classpathDependencyExcludes>
               </classpathDependencyExcludes>
+              <properties>
```

Comment:
Also the profiles are almost identical, except for the groups property. In driver-extras I tried to factorize all the settings out of the profiles into the main declaration, and only set the `test.groups` property in each profile.